dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -131,14 +132,16 @@ jettison/1.5.4//jettison-1.5.4.jar
 jetty-util-ajax/9.4.52.v20230823//jetty-util-ajax-9.4.52.v20230823.jar
 jetty-util/9.4.52.v20230823//jetty-util-9.4.52.v20230823.jar
 jline/2.14.6//jline-2.14.6.jar
+jline/3.22.0//jline-3.22.0.jar
+jna/5.13.0//jna-5.13.0.jar
```

Comment:
why do they become runtime deps now?

And since both jline 2 and 3 are present at the classpath, is it safe?

Reply:
Yes, I saw this dependency before, It is a cascading dependency of Scala 2.13. I think it is safe because the package name is different

jline2 is `jline`

<img width="467" alt="image" src="https://github.com/apache/spark/assets/1475305/ea1d77d6-c1ef-401e-a679-9433872ff712">

jline3 is `org.jline`

<img width="439" alt="image" src="https://github.com/apache/spark/assets/1475305/60513fca-bbbd-4192-bb21-ade59891db03">

Reply:
it's fine then, thanks for checking